tests/system/runtimes/test_databricks.py

```diff
@@ -98,7 +99,8 @@ def _abort_run(self):
 
     def _check_artifacts(self, paths_dict):
         artifacts = self.project.list_artifacts().to_objects()
-        assert len(artifacts) == len(paths_dict)
+        #  +1 because of metadata artifact
```

Comment:
```suggestion
        # +1 because of metadata artifact
```
Not sure what's up with the double space in the comments.